manifests/server.pp

```diff
@@ -29,36 +29,13 @@
   Boolean $pam             = simplib::lookup('simp_options::pam', { 'default_value' => false }),
   Boolean $clamav          = simplib::lookup('simp_options::clamav', { 'default_value' => false }),
   Boolean $selinux         = simplib::lookup('simp_options::selinux', { 'default_value' => false }),
-  Boolean $auditd          = simplib::lookup('simp_options::auditd', { 'default_value' => false })
-) {
-
-  include '::aide'
-  include '::at'
-  include '::chkrootkit'
-  include '::cron'
-  include '::incron'
-  include '::issue'
-  include '::nsswitch'
-  include '::ntpd'
-  include '::pam::access'
-  include '::pam::wheel'
-  include '::pupmod'
-  include '::resolv'
-  include '::ssh'
-  include '::sudosh'
-  include '::svckill'
-  include '::swap'
-  include '::timezone'
-  include '::tuned'
-  include '::useradd'
-  include '::simp::admin'
-  include '::simp::base_apps'
-  include '::simp::base_services'
-  include '::simp::kmod_blacklist'
-  include '::simp::mountpoints'
-  include '::simp::server::rsync_shares'
-  include '::simp::sysctl'
+  Boolean $auditd          = simplib::lookup('simp_options::auditd', { 'default_value' => false }),
+  String $scenario         = $simp::scenario,
+  Array $classes           = [],
+  Hash $scenario_map,
+) inherits simp {
 
+  simp::classifier($scenario, $scenario_map, $classes)
   if $clamav  { include '::clamav' }
```

Comment:
so I don't forget, 6.1 might be a good chance to look at making simp_options::* toggle classes in simp::classifier. Just for the lulz